index.html

```diff
@@ -60,34 +60,43 @@
             </div>
             <div class="info">
                 <p>
-                    Version: Age of Empires II DE Update <a href="https://www.ageofempires.com/news/aoe2de-update-36906/" target="_blank" title="Changelog">36906</a>
+                    Version: Age of Empires II DE Update <a href="https://www.ageofempires.com/news/aoe2de-update-36906/" target="_blank" title="Changelog" rel="nofollow">36906</a>
                 </p>
                 <p>
-                    Contribute on <a href="https://github.com/SiegeEngineers/aoe2techtree" target="_blank">GitHub</a>
-                </p>
-                <p>
-                    Made by hszemi, Anda, exterkamp, paulirish,<br>
-                    with thanks to Jineapple, TriRem, pip, and NkoDragaš<br>
-                    Item Metadata (cost, HP etc.) taken from <a href="https://github.com/HSZemi/aoe2dat" target="_blank">aoe2dat</a>
-                </p>
-                <p>
-                    Age of Empires II &copy; Microsoft Corporation.</br>
-                    <b>aoe2techtree</b> was created under Microsoft's "<a href="https://www.xbox.com/en-us/developers/rules">Game Content Usage Rules</a>" using assets
+                    Age of Empires II &copy; Microsoft Corporation.<br>
+                    <b>aoe2techtree</b> was created under Microsoft's "<a href="https://www.xbox.com/en-us/developers/rules" rel="nofollow">Game Content Usage Rules</a>" using assets
                     from Age of Empires II, and it is not endorsed by or affiliated with Microsoft.
                 </p>
+                <details>
+                    <summary>Credits</summary>
```

Comment:
Do we want to "hide" the credits like this? :thinking:

Reply:
My intention was to reduce the text on that side bar 😄

Reply:
Also, we may need some space for the new feature that I was planning to submit a PR for: Color Key / Legend
 
![image](https://user-images.githubusercontent.com/13775/81175901-f039c780-8fc1-11ea-96d8-15d57399ce97.png)

Reply:
ayayay

Reply:
So what is the final decision on it? This is fine or should I revert back the credits section to how it was earlier?

> Version: Age of Empires II DE Update 36906
> 
> Contribute on GitHub
> 
> Made by hszemi, Anda, exterkamp, paulirish,
> with thanks to Jineapple, TriRem, pip, and NkoDragaš
> Item Metadata (cost, HP etc.) taken from aoe2dat
> 
> Age of Empires II © Microsoft Corporation.
> aoe2techtree was created under Microsoft's "Game Content Usage Rules" using assets from Age of Empires II, and it is not endorsed by or affiliated with Microsoft.

Reply:
Let's keep the old version for now and revisit this once the legend is suggested

Reply:
Done!